src/crimson/net/SocketMessenger.h

```diff
@@ -152,6 +152,17 @@ class SocketMessenger final : public Messenger {
   Interceptor *interceptor = nullptr;
 #endif
 
+  seastar::future<> mark_down(const entity_addr_t& a) final {
+    auto conn = lookup_conn(a);
+    if (conn) {
+      return seastar::smp::submit_to(
+	conn->get_shard_id(),
+	[conn] {
+	conn->mark_down();
+	return seastar::now();
+      });
+    }
```

Comment:
missing return `seastar::now()`

```
src/crimson/net/Messenger.cc:5:
/home/jenkins-build/build/workspace/ceph-pull-requests/src/crimson/net/SocketMessenger.h:165:3: error: non-void function does not return a value in all control paths [-Werror,-Wreturn-type]
  }
```

Reply:
@xxhdx1985126, this fails makecheck

Reply:
Sorry for this inconvenience, just fixed.